dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -129,8 +233,48 @@ export const Columns: React.FC<{
 				column={column}
 				key={column.title.toLowerCase()}
 				index={i}
+				showLine={i !== nav.pillars.length - 1}
 			/>
 		))}
+		<li>
+			<form css={searchBar} action="https://www.google.co.uk/search">
+				<input
+					css={searchInput}
+					type="text"
+					name="q"
+					placeholder="Search"
+					data-link-name="nav2 : search"
+				/>
```

Comment:
These types of components can be imported from Source. We use the design system for these things as it means we get production ready, tested, accessible components out of the box.

You can see more [here](https://theguardian.design/2a1e5182b/p/77ee17-core-components) but there's components for [inputs](https://theguardian.design/2a1e5182b/p/097455-text-input-field), [labels](https://theguardian.design/2a1e5182b/p/40151e-label), etc.

Reply:
(Where we want to override Source styles there's a `cssOverrides` prop)